File: AP Computer Science A/Programming Projects/Physics Calculators/src/SHM_Springs.java
import java.util.Scanner;
public class SHM_Springs {
    public static void main(String[] args) {
        //for springs
        Scanner sub = new Scanner(System.in);
        System.out.println("What is your k value?");
        double kVal = sub.nextDouble();
        System.out.println("What is your mass?");
        double mass = sub.nextDouble();
        System.out.println((2*3.14159265*(kVal/mass)));

    }
}

File: AP Computer Science A/Programming Projects/Physics Calculators/src/SHM Springs.py
print("Are you using springs or pendulums?")  # intro-project
sOrp = input()
if sOrp == "s":
    print("What is your mass value?")
    mass = input()
    print("What is your k value?")
    k = input()
    print("Period is:")
    print(2*3.14*(int(mass)/int(k))**1/2)
elif sOrp == "p":
    print("What is your L value?")
    l = input()
    print("Period is:")
    print(2*3.14*(int(l)/int(9.8))**1/2)


